extensions/WSLExtension/Models/WslComputeSystem.cs

```diff
@@ -81,7 +81,12 @@ public ComputeSystemOperations SupportedOperations
         _distribution = distribution;
         _wslManager = wslManager;
         DisplayName = distribution.FriendlyName;
-        _curPackage = _packageHelper.GetPackageFromPackageFamilyName(_distribution.PackageFamilyName!);
+
+        // Only get the package information if it has a package family name
+        if (_distribution.PackageFamilyName != null)

```

Comment:
nit: is this if statement needed here or the changes in the helper class should cover all cases?

Reply:
same below